src/platform_impl/linux/x11/mod.rs

```diff
@@ -261,6 +268,9 @@ impl<T: 'static> EventLoop<T> {
         // Create a channel for handling redraw requests.
         let (redraw_sender, redraw_channel) = channel();
 
+        // Create a channel for sending activation tokens.
+        let (activation_token_sender, activation_token_channel) = channel();
```

Comment:
Can't we merge it with some other delivery channel?

Reply:
I tried merging it with the user event delivery channel, but the problem is that this channel needs to be used by the `Window`, which can't be generic.